### PR TITLE
Feature/merge product pricings to products table

### PR DESCRIPTION
**Pull request by achmadfatoni**
i put back Accessor and Mutator for expiry date, becasue i think it is the best way to manipulate data when set and get from database

### CODE REVIEW

File: src/Klsandbox/OrderModel/Models/Product.php [hunk 1]
```diff
@@ -330,4 +197,43 @@ public static function findByName($name, $strict = true)
...
+                $price = $this->price_east;
+                $delivery = $this->delivery_east;
+            } else {
+                $price = $this->price;
+                $delivery = $this->delivery;
+            }
+        }
+    }
+
+    //Accessor
```
Comment (mrwaim) [inline src/Klsandbox/OrderModel/Models/Product.php]: I removed this already, why did you add it back?

Comment (achmadfatoni) [inline src/Klsandbox/OrderModel/Models/Product.php]: i put back Accessor and Mutator for expiry date, becasue i think it is the best way to manipulate data when set and get from database.

i need to get data in d/m/Y for for edit form, and for create new data format to Y-m-d.

do i need to do it manually?

File: src/Klsandbox/OrderModel/Models/OrderItem.php [hunk 2]
```diff
@@ -84,14 +87,14 @@ public function activeBonuses()
...
-     * Relationship with `product_pricings` table.
-     *
-     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
-     */
+    public function product()
+    {
+        return $this->belongsTo(Product::class);
+    }
+
     public function productPricing()
```
Comment (mrwaim) [inline src/Klsandbox/OrderModel/Models/OrderItem.php]: remove
